nss_cache/sources/ldapsource.py

```diff
@@ -566,10 +579,13 @@ class GroupUpdateGetter(UpdateGetter):
 
   def __init__(self,conf):
     super(GroupUpdateGetter, self).__init__()
-    if conf.has_key('rfc2307bis') and conf['rfc2307bis']:
+    self.conf = conf
+    if 'rfc2307bis' in conf and conf['rfc2307bis']:
```

Comment:
That's good right?  It's not supposed to be default.  ldap_rfc2307bis = 1 must be set in the config.